docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -425,7 +425,7 @@ The configuration above generates the following sidebar:
 				{ label: 'Components', link: '/guides/components/' },
 				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
 				{
-					label: 'advanced',
+					label: 'Advanced',
```

Comment:
This is assuming the same filesystem structure as the one in the autogenerated groups section linked line 402 which means that in this case, this would be a 1:1 match to an `advanced/` directory.